packages/e2e-tests/specs/plugins/meta-attribute-block.test.js

```diff
@@ -40,9 +40,12 @@ describe( 'Block with a meta attribute', () => {
 		await insertBlock( 'Test Meta Attribute Block' );
 		await page.keyboard.type( 'Meta Value' );
 
-		const persistedValues = await page.evaluate( () => Array.from( document.querySelectorAll( '.my-meta-input' ) ).map( ( input ) => input.value ) );
-		persistedValues.forEach( ( val ) => {
-			expect( val ).toBe( 'Meta Value' );
+		const inputs = await page.$$( '.my-meta-input' );
+		await inputs.forEach( async ( input ) => {
+			// Clicking the input selects the block,
+			// and selecting the block enables the sync data mode.
```

Comment:
> enables the sync data mode

It wouldn't be clear to the reader why this matters. Something like:

>[...], as otherwise the asynchronous rerendering of unselected blocks may cause the input to have not yet been updated for the other blocks.